### DISCUSSION TITLE
When wiznet5K library use, it occurs the problem.

### DISCUSSION THREAD

**Discussion by irinakim12** [micropython/micropython#10157] [Libraries & Drivers]
https://docs.micropython.org/en/latest/library/network.WIZNET5K.html?highlight=wiznet5k#network.WIZNET5K
If you follow the contents of the docs site, it does not work normally,

```py
wiznet5k.active(True)
``` 
 should be entered.

I've already sent a pullrequest about that, but I don't know if it's being processed or if I need to do something.

https://github.com/micropython/micropython/pull/9956

**Comment by jimmo** [comment]
@irinakim12 The reviewers just have a lot to do! We will reply to your PR soon.
